src/NuGet.Services.SearchService/App_Start/WebApiConfig.cs

```diff
@@ -99,6 +107,11 @@ private static IDependencyResolver GetDependencyResolver(HttpConfiguration confi
             builder.Populate(services);
             builder.AddAzureSearch();
 
+            var loggerConfiguration = LoggingSetup.CreateDefaultLoggerConfiguration(withConsoleLogger: false);
```

Comment:
Why did we replace `services.AddLogging` with this?
Does `AddLogging` not pick up the Application Insights instrumentation?

Reply:
AddLogging doesn't use the Serilog enriched LoggerFactory. Gallery does not use AddLogging and I am basically aligning with that